docs/hugo/content/contributing/generator-overview.md

```diff
@@ -0,0 +1,56 @@
+# Code Generator Overview
+
+Core to Azure Service Operator (ASO) v2 is our code generator. This consumes ARM JSON Schema and Swagger specifications and generates code for each desired resource that works with our generic operator reconciler.
+
+## Code Structure
+
```

Comment:
A link to the actual location in the tree you're talking about might be good

Reply:
Adding links is a good idea - I'll do those in a separate PR, I've created #2077 to make sure I don't forget.